### PR TITLE
Use smaller LLMQs in regtest

### PR DESCRIPTION
**Pull request by codablock**
Use quorums of size 3 with threshold 2 for most integration tests. Only a few need a larger one due to the way testing is done.

I assume that individual runtime of tests won't improve much. My hope/assumption here is that CPU load on Travis/Gitlab can be reduced dramatically, so that all tests benefit while executed in parallel.

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -892,3 +899,8 @@ void UpdateDevnetLLMQChainLocks(Consensus::LLMQType llmqType)
 {
     globalChainParams->UpdateLLMQChainLocks(llmqType);
 }
+
+void UpdateLLMQTestParams(int size, int threshold)
+{
+    globalChainParams->UpdateLLMQTestParams(size, threshold);
+}
```
Comment (UdjinM6) [inline src/chainparams.cpp]: nit: new line at EOF
```suggestion
}

```

**Comment by codablock** [comment]
As I assumed, total runtime on Gitlab benefited from this. Tests on develop:
```
TEST                           | STATUS    | DURATION
abandonconflict.py             | P Passed  | 5 s
addressindex.py                | P Passed  | 13 s
assumevalid.py                 | P Passed  | 14 s
bip65-cltv-p2p.py              | P Passed  | 9 s
bip68-112-113-p2p.py           | P Passed  | 32 s
bip68-sequence.py              | P Passed  | 15 s
bip9-softforks.py              | P Passed  | 432 s
bipdersig-p2p.py               | P Passed  | 7 s
bitcoin_cli.py                 | P Passed  | 2 s
blockchain.py                  | P Passed  | 5 s
decodescript.py                | P Passed  | 1 s
dip3-deterministicmns.py       | P Passed  | 108 s
dip4-coinbasemerkleroots.py    | P Passed  | 147 s
disablewallet.py               | P Passed  | 1 s
disconnect_ban.py              | P Passed  | 1 s
example_test.py                | P Passed  | 2 s
feature_shutdown.py            | P Passed  | 1 s
forknotify.py                  | P Passed  | 2 s
fundrawtransaction-hd.py       | P Passed  | 26 s
fundrawtransaction.py          | P Passed  | 18 s
getblocktemplate_longpoll.py   | P Passed  | 66 s
getchaintips.py                | P Passed  | 5 s
httpbasics.py                  | P Passed  | 1 s
import-rescan.py               | P Passed  | 4 s
importmulti.py                 | P Passed  | 4 s
importprunedfunds.py           | P Passed  | 1 s
invalidateblock.py             | P Passed  | 6 s
invalidblockrequest.py         | P Passed  | 2 s
invalidtxrequest.py            | P Passed  | 2 s
keypool-hd.py                  | P Passed  | 3 s
keypool-topup.py               | P Passed  | 9 s
keypool.py                     | P Passed  | 3 s
listsinceblock.py              | P Passed  | 4 s
listtransactions.py            | P Passed  | 4 s
llmq-chainlocks.py             | P Passed  | 136 s
llmq-dkgerrors.py              | P Passed  | 174 s
llmq-is-cl-conflicts.py        | P Passed  | 75 s
llmq-is-retroactive.py         | P Passed  | 279 s
llmq-signing.py                | P Passed  | 80 s
llmq-simplepose.py             | P Passed  | 113 s
maxuploadtarget.py             | P Passed  | 44 s
mempool_limit.py               | P Passed  | 7 s
mempool_packages.py            | P Passed  | 6 s
mempool_persist.py             | P Passed  | 7 s
mempool_reorg.py               | P Passed  | 4 s
mempool_resurrect_test.py      | P Passed  | 2 s
mempool_spendcoinbase.py       | P Passed  | 2 s
merkle_blocks.py               | P Passed  | 4 s
minchainwork.py                | P Passed  | 5 s
mining.py                      | P Passed  | 4 s
multi_rpc.py                   | P Passed  | 3 s
multikeysporks.py              | P Passed  | 40 s
multiwallet.py                 | P Passed  | 33 s
net.py                         | P Passed  | 1 s
nulldummy.py                   | P Passed  | 3 s
p2p-acceptblock.py             | P Passed  | 5 s
p2p-compactblocks.py           | P Passed  | 17 s
p2p-fingerprint.py             | P Passed  | 8 s
p2p-fullblocktest.py           | P Passed  | 580 s
p2p-instantsend.py             | P Passed  | 41 s
p2p-leaktests.py               | P Passed  | 6 s
p2p-mempool.py                 | P Passed  | 1 s
p2p-timeouts.py                | P Passed  | 62 s
p2p-versionbits-warning.py     | P Passed  | 11 s
preciousblock.py               | P Passed  | 2 s
prioritise_transaction.py      | P Passed  | 5 s
privatesend.py                 | P Passed  | 1 s
proxy_test.py                  | P Passed  | 1 s
rawtransactions.py             | P Passed  | 16 s
receivedby.py                  | P Passed  | 9 s
reindex.py                     | P Passed  | 10 s
resendwallettransactions.py    | P Passed  | 1 s
rest.py                        | P Passed  | 11 s
rpc_getblockstats.py           | P Passed  | 1 s
rpcbind_test.py                | o Skipped | 0 s
rpcnamedargs.py                | P Passed  | 3 s
sendheaders.py                 | P Passed  | 25 s
signmessages.py                | P Passed  | 1 s
signrawtransactions.py         | P Passed  | 1 s
smartfees.py                   | P Passed  | 179 s
spentindex.py                  | P Passed  | 14 s
sporks.py                      | P Passed  | 8 s
timestampindex.py              | P Passed  | 4 s
txindex.py                     | P Passed  | 6 s
txn_clone.py                   | P Passed  | 3 s
txn_clone.py --mineblock       | P Passed  | 3 s
txn_doublespend.py             | P Passed  | 3 s
txn_doublespend.py --mineblock | P Passed  | 2 s
uptime.py                      | P Passed  | 1 s
wallet-accounts.py             | P Passed  | 6 s
wallet-dump.py                 | P Passed  | 72 s
wallet-encryption.py           | P Passed  | 7 s
wallet-hd.py                   | P Passed  | 24 s
wallet.py                      | P Passed  | 32 s
walletbackup.py                | P Passed  | 124 s
zapwallettxes.py               | P Passed  | 4 s
zmq_test.py                    | P Passed  | 4 s
ALL                            | P Passed  | 3291 s (accumulated) 
Runtime: 831 s
```

This branch:
```
TEST                           | STATUS    | DURATION
abandonconflict.py             | P Passed  | 9 s
addressindex.py                | P Passed  | 52 s
assumevalid.py                 | P Passed  | 13 s
bip65-cltv-p2p.py              | P Passed  | 11 s
bip68-112-113-p2p.py           | P Passed  | 22 s
bip68-sequence.py              | P Passed  | 16 s
bip9-softforks.py              | P Passed  | 376 s
bipdersig-p2p.py               | P Passed  | 7 s
bitcoin_cli.py                 | P Passed  | 0 s
blockchain.py                  | P Passed  | 10 s
decodescript.py                | P Passed  | 1 s
dip3-deterministicmns.py       | P Passed  | 104 s
dip4-coinbasemerkleroots.py    | P Passed  | 78 s
disablewallet.py               | P Passed  | 1 s
disconnect_ban.py              | P Passed  | 6 s
example_test.py                | P Passed  | 2 s
feature_shutdown.py            | P Passed  | 2 s
forknotify.py                  | P Passed  | 2 s
fundrawtransaction-hd.py       | P Passed  | 35 s
fundrawtransaction.py          | P Passed  | 20 s
getblocktemplate_longpoll.py   | P Passed  | 66 s
getchaintips.py                | P Passed  | 4 s
httpbasics.py                  | P Passed  | 1 s
import-rescan.py               | P Passed  | 4 s
importmulti.py                 | P Passed  | 14 s
importprunedfunds.py           | P Passed  | 1 s
invalidateblock.py             | P Passed  | 7 s
invalidblockrequest.py         | P Passed  | 3 s
invalidtxrequest.py            | P Passed  | 2 s
keypool-hd.py                  | P Passed  | 4 s
keypool-topup.py               | P Passed  | 4 s
keypool.py                     | P Passed  | 4 s
listsinceblock.py              | P Passed  | 3 s
listtransactions.py            | P Passed  | 3 s
llmq-chainlocks.py             | P Passed  | 107 s
llmq-dkgerrors.py              | P Passed  | 110 s
llmq-is-cl-conflicts.py        | P Passed  | 54 s
llmq-is-retroactive.py         | P Passed  | 252 s
llmq-signing.py                | P Passed  | 84 s
llmq-simplepose.py             | P Passed  | 108 s
maxuploadtarget.py             | P Passed  | 44 s
mempool_limit.py               | P Passed  | 16 s
mempool_packages.py            | P Passed  | 5 s
mempool_persist.py             | P Passed  | 10 s
mempool_reorg.py               | P Passed  | 3 s
mempool_resurrect_test.py      | P Passed  | 1 s
mempool_spendcoinbase.py       | P Passed  | 1 s
merkle_blocks.py               | P Passed  | 8 s
minchainwork.py                | P Passed  | 5 s
mining.py                      | P Passed  | 3 s
multi_rpc.py                   | P Passed  | 3 s
multikeysporks.py              | P Passed  | 35 s
multiwallet.py                 | P Passed  | 3 s
net.py                         | P Passed  | 2 s
nulldummy.py                   | P Passed  | 3 s
p2p-acceptblock.py             | P Passed  | 6 s
p2p-compactblocks.py           | P Passed  | 17 s
p2p-fingerprint.py             | P Passed  | 8 s
p2p-fullblocktest.py           | P Passed  | 458 s
p2p-instantsend.py             | P Passed  | 70 s
p2p-leaktests.py               | P Passed  | 6 s
p2p-mempool.py                 | P Passed  | 1 s
p2p-timeouts.py                | P Passed  | 62 s
p2p-versionbits-warning.py     | P Passed  | 10 s
preciousblock.py               | P Passed  | 1 s
prioritise_transaction.py      | P Passed  | 7 s
privatesend.py                 | P Passed  | 2 s
proxy_test.py                  | P Passed  | 3 s
rawtransactions.py             | P Passed  | 12 s
receivedby.py                  | P Passed  | 9 s
reindex.py                     | P Passed  | 2 s
resendwallettransactions.py    | P Passed  | 2 s
rest.py                        | P Passed  | 6 s
rpc_getblockstats.py           | P Passed  | 1 s
rpcbind_test.py                | o Skipped | 0 s
rpcnamedargs.py                | P Passed  | 2 s
sendheaders.py                 | P Passed  | 27 s
signmessages.py                | P Passed  | 1 s
signrawtransactions.py         | P Passed  | 2 s
smartfees.py                   | P Passed  | 208 s
spentindex.py                  | P Passed  | 40 s
sporks.py                      | P Passed  | 8 s
timestampindex.py              | P Passed  | 10 s
txindex.py                     | P Passed  | 5 s
txn_clone.py                   | P Passed  | 2 s
txn_clone.py --mineblock       | P Passed  | 2 s
txn_doublespend.py             | P Passed  | 2 s
txn_doublespend.py --mineblock | P Passed  | 2 s
uptime.py                      | P Passed  | 1 s
wallet-accounts.py             | P Passed  | 3 s
wallet-dump.py                 | P Passed  | 73 s
wallet-encryption.py           | P Passed  | 7 s
wallet-hd.py                   | P Passed  | 23 s
wallet.py                      | P Passed  | 25 s
walletbackup.py                | P Passed  | 101 s
zapwallettxes.py               | P Passed  | 9 s
zmq_test.py                    | P Passed  | 7 s
ALL                            | P Passed  | 2987 s (accumulated) 
Runtime: 754 s
```

**Comment by UdjinM6** [comment]
Needs rebase

**Comment by codablock** [comment]
test failures are due to #3272. I'll rebase this PR after the fix got merged

**Pull request merged**